atos/modules/OpenScenarioGateway/openscenariogateway.py

```diff
@@ -87,23 +93,38 @@ def init_callback(self, msg):
     def story_board_element_state_change_callback(self, story_board_element):
         if (
             story_board_element.name in self.follow_traj_to_obj_name.keys()
-            and story_board_element.state == 2
-        ):  # 2 is a running state
-
-            # Iterate through active objects to send the start command for the target objects
-            for object_id, object in self.active_objects.items():
-                target_object_name = self.follow_traj_to_obj_name[
-                    story_board_element.name
-                ]
-                if object.name in target_object_name:
-
-                    self.get_logger().info(
-                        f"Starting object {object.name} with id {object_id}"
-                    )
-                    start_object_msg = atos_interfaces.msg.ObjectTriggerStart()
-                    start_object_msg.id = object_id
-                    self.start_object_pub_.publish(start_object_msg)
-                    break
+            and story_board_element.state == 2  # 2 is a running state
```

Comment:
Is it possible to make the 2 an enum for added clarity, and if we're going to use more states in the future?

Reply:
Done